crates/node/core/src/args/network.rs

```diff
@@ -219,14 +219,11 @@ impl NetworkArgs {
 
     /// Resolve all trusted peers at once
     pub async fn resolve_trusted_peers(&self) -> eyre::Result<Vec<NodeRecord>> {
```

Comment:
change the eror variant, so that this can be returned directly without doing `?`